### PR TITLE
Renaming space heater

### PR DESCRIPTION
**Pull request by NortonDK**
Changed the name and description of the space heater(now space A/C), to better show that it can also cool

### CODE REVIEW

**Comment by NortonDK** [comment]
!review

File: code/game/machinery/spaceheater.dm [hunk 1]
```diff
@@ -3,8 +3,8 @@
 	density = 1
 	icon = 'icons/obj/atmos.dmi'
 	icon_state = "sheater0"
-	name = "space heater"
-	desc = "Made by Space Amish using traditional space techniques, this heater is guaranteed not to set the station on fire."
+	name = "space A/C"
```
Comment (MarinaGryphon) [inline code/game/machinery/spaceheater.dm]: Make sure you've tested to see if the slash character breaks anything. It shouldn't, but who knows, with BYOND.

Comment (NortonDK) [reply code/game/machinery/spaceheater.dm]: I did and it worked fine :)

Comment (NortonDK) [reply code/game/machinery/spaceheater.dm]: Just did a test and it works fine

**Comment by Arrow768** [comment]
The name sounds a bit clunky, but its good enough to merge

**Pull request merged**